opm/simulators/timestepping/SimulatorReport.cpp

```diff
@@ -73,15 +73,14 @@ namespace Opm
 
     void SimulatorReportSingle::reportStep(std::ostringstream& ss) const
     {
-        ss << "Time step summary: ";
         if (total_well_iterations != 0) {
-            ss << "well its = " << std::setw(2) << total_well_iterations << ", ";
+            ss << "Well its=" << std::setw(2) << total_well_iterations << ", ";
```

Comment:
I must admit that the missing spaces around `=` look ugly to me. Could you post a "before and after" example? I think setw(2) for various times might be too low for somewhat large cases, could we go with setw(3)? I assume the goal is to reduce the total width of the message. To that end I propose dropping the `linearizations` printout here (keeping it only on the final end-of-run printout). That would gain many characters!